evalml/tests/automl_tests/test_automl.py

```diff
@@ -2114,6 +2188,23 @@ def fit(self, *args, **kwargs):
         )
 
 
+def test_time_series_regression_with_parameters(ts_data):
```

Comment:
Can we not patch this test to make it run faster? We are just making sure that the problem configuration is set properly, right?

Reply:
`search` is actually not even required here, good catch!